engine/platform/src/platform_window_null.cpp

```diff
@@ -118,6 +118,9 @@ namespace dmPlatform
     void PollEvents(HWindow window)
     {}
 
+    void SwapBuffers(HWindow window)
+    {}
+
```

Comment:
We should add a test to make sure it links (like we have for physics_null library)